lib/git_ops/git.ex

```diff
@@ -70,18 +70,8 @@ defmodule GitOps.Git do
   @spec hooks_path(Git.Repository.t()) :: String.t() | no_return
   def hooks_path(repo) do
     case Git.config(repo, ["core.hookspath"]) do
-      {:error, %Git.Error{message: "", code: 1}} ->
-        # no custom config for core.hookspath
-        if File.dir?(@default_githooks_path) do
-          @default_githooks_path
-        else
-          raise """
-          Could not find the default git hooks path #{inspect(@default_githooks_path)}. Is this a git repo?
-          """
-        end
-
-      {:error, %Git.Error{message: message}} ->
-        raise message
+      {:error, error} ->
+        handle_hooks_path_error(error)
```

Comment:
Is this just a stylistic change? The result of this code looks the same to me, just want to make sure.

Reply:
Yeah, dialyzer was complaining because the default spec for an exception doesn't guarantee a message field, and the `git_cli` library doesn't override the default spec: https://github.com/danhper/elixir-git-cli/blob/master/lib/git/error.ex#L1